scrunch/datasets.py

```diff
@@ -1456,15 +1454,11 @@ def __repr__(self):
     def __str__(self):
         return self.name
 
-    _categories = None
-
     @property
     def categories(self):
         if self.resource.body['type'] not in self.CATEGORICAL_TYPES:
             raise TypeError("Variable of type %s do not have categories" % self.resource.body.type)
-        if self._categories is None:
-            self._categories = CategoryList(self.resource)
-        return self._categories
+        return CategoryList._from(self.resource)
```

Comment:
Watch for a regression here, I had a test that made sure that you didn't return a new CategoryList instance each time you accessed the property. That `if` cached the instance once and reused it. With your change we always instantiate and init a new one. Is this deliberate?